starfyre/__main__.py

```diff
@@ -9,34 +9,87 @@
 import click
 import importlib.resources as pkg_resources
 
-
 def write_js_file(path):
     dist_path = Path(path) / "dist"
     dist_path.mkdir(exist_ok=True)
     js_store = pkg_resources.path("starfyre.js", "store.js")
     shutil.copy(str(js_store), path + "/dist/store.js")
-    
-def create_main_file(path):
-    """
-    Creates a main file in the build directory.
-    
-    This file will be used to run the project.
-     Every starfyre build will have an `__init__.py` file in the build directory.
-    And the `__init__.py` file will have a component that will render the root component. It will be named `app`.
 
-    You can have a look at the `test-application/build/__init__.py` file to see what it looks like.
+def build_routes_output(generated_routes, path):
```

Comment:
Hey @DeleMike 👋 

Can you please add a docstring here?

Reply:
Done.